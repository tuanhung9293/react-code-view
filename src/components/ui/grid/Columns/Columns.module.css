.base{
  box-sizing: border-box;
  display: flex;
  flex-wrap: wrap;
  justify-content: space-between;
  margin: 0 -10px;
}

.center{
  justify-content: center;
}