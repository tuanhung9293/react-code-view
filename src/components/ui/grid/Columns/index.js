import React from 'react'
import classNames from 'classnames'
import styles from './Columns.module.css'

function Columns({children, center}) {
  const className = classNames({
    [styles.base]: true,
    [styles.center]: center
  })
  return <div className={className}>{children}</div>
}

export default Columns