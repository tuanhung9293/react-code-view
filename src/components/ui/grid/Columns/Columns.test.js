import React from 'react'
import {shallow} from 'enzyme'
import Columns from './'

it('renders children', () => {
  expect(shallow(
    <Columns>
      <p>I feel like a child</p>
    </Columns>
  )).toMatchSnapshot()
})
