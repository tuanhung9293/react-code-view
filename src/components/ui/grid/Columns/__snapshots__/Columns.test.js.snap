// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders children 1`] = `
<div
  className="base"
>
  <p>
    I feel like a child
  </p>
</div>
`;
